packages/neuron-wallet/src/models/chain-info.ts

```diff
@@ -0,0 +1,29 @@
+export default class ChainInfo {
+  private static instance: ChainInfo
+
+  static getInstance(): ChainInfo {
+    if (!ChainInfo.instance) {
+      ChainInfo.instance = new ChainInfo()
+    }
+
+    return ChainInfo.instance
+  }
+
+  private chain: string = ''
+
+  public setChain = (chain: string) => {
+    this.chain = chain
+  }
+
+  public getChain = (): string => {
+    return this.chain
+  }
+
+  public isMainnet = (): boolean => {
```

Comment:
This could be as simple as `return this.chain === 'ckb'`.

Reply:
updated